Detectors/GLOQC/src/MatchITSTPCQC.cxx

```diff
@@ -375,9 +385,50 @@ bool MatchITSTPCQC::init()
     }
   }
 
+  // log binning for pT for K0s
+  const Int_t nbinsPtK0 = 10;
+  const Double_t xminPtK0 = 0.01;
+  const Double_t xmaxPtK0 = 20;
+  Double_t* xbinsPtK0 = new Double_t[nbinsPtK0 + 1];
+  Double_t xlogminPtK0 = TMath::Log10(xminPtK0);
+  Double_t xlogmaxPtK0 = TMath::Log10(xmaxPtK0);
+  Double_t dlogxPtK0 = (xlogmaxPtK0 - xlogminPtK0) / nbinsPtK0;
+  for (int i = 0; i <= nbinsPtK0; i++) {
+    Double_t xlogPtK0 = xlogminPtK0 + i * dlogxPtK0;
+    xbinsPtK0[i] = TMath::Exp(TMath::Log(10) * xlogPtK0);
+  }
+  // the other bins
+  const Int_t nbinsMassK0 = 100;
+  Double_t* ybinsMassK0 = new Double_t[nbinsMassK0 + 1];
+  Double_t yminMassK0 = 0.4;
+  Double_t ymaxMassK0 = 0.6;
+  Double_t dyMassK0 = (ymaxMassK0 - yminMassK0) / nbinsMassK0;
+  for (int i = 0; i <= nbinsMassK0; i++) {
+    ybinsMassK0[i] = yminMassK0 + i * dyMassK0;
+  }
+  const Int_t nbinsMultK0pp = mNBinsTPCOccpp;
+  Double_t* zbinsMultK0pp = new Double_t[nbinsMultK0pp + 1];
+  Double_t zminMultK0pp = mMinTPCOccpp;
+  Double_t zmaxMultK0pp = mMaxTPCOccpp;
+  Double_t dzMultK0pp = (zmaxMultK0pp - zminMultK0pp) / nbinsMultK0pp;
+  for (int i = 0; i <= nbinsMultK0pp; i++) {
+    zbinsMultK0pp[i] = zminMultK0pp + i * dzMultK0pp;
+  }
+
+  const Int_t nbinsMultK0PbPb = mNBinsTPCOccPbPb;
+  Double_t* zbinsMultK0PbPb = new Double_t[nbinsMultK0PbPb + 1];
+  Double_t zminMultK0PbPb = mMinTPCOccPbPb;
+  Double_t zmaxMultK0PbPb = mMaxTPCOccPbPb;
+  Double_t dzMultK0PbPb = (zmaxMultK0PbPb - zminMultK0PbPb) / nbinsMultK0PbPb;
+  for (int i = 0; i <= nbinsMultK0PbPb; i++) {
+    zbinsMultK0PbPb[i] = zminMultK0PbPb + i * dzMultK0PbPb;
+  }
+
```

Comment:
one should `delete[] ` axes binning arrays booked with `new`.

Reply:
Done.